test/regressions/index.test.js

```diff
@@ -1,10 +1,13 @@
+import * as url from 'url';
 import * as path from 'path';
 import * as fse from 'fs-extra';
 import * as playwright from 'playwright';
 
+const currentDirectory = url.fileURLToPath(new URL('.', import.meta.url));
+
 async function main() {
   const baseUrl = 'http://localhost:5001';
-  const screenshotDir = path.resolve(__dirname, './screenshots/chrome');
+  const screenshotDir = path.resolve(currentDirectory, './screenshots/chrome');
```

Comment:
Can we directly use `import.meta.dirname` alias here as per the docs: https://github.com/javiertury/babel-plugin-transform-import-meta?tab=readme-ov-file#importmetadirname.
```suggestion
  const screenshotDir = path.resolve(import.meta.dirname, './screenshots/chrome');
```

Reply:
Ah, good find. Technically yes, the plugin supports it. But if we want to be able to switch to native Node.js ESM, then only when Node.js supports it. Looks like that got [added](https://nodejs.org/api/esm.html#importmetadirname) in v20.10.0.

We're running our [CI](https://github.com/mui/material-ui/blob/ca280812f5edc6a21301dd770ff617d0b3bec076/.circleci/config.yml#L48) on 20.17, so that should be fine, but I know we [don't force](https://github.com/mui/material-ui/blob/ca280812f5edc6a21301dd770ff617d0b3bec076/package.json#L204) a node version for maintainers. We use the `currentDirectory` method all over the place. Personally I'd leave it as is now and change all instances once it's a more stable feature and we add node 20 to the engines field.

Reply:
@Janpot Sounds good.